### PR TITLE
Add Traefik guide to Swarm practice

### PR DESCRIPTION
**Pull request by yvidalv**
Add Traefik guide to Swarm practice

### CODE REVIEW

File: cursos/docker/00_actividades/06_modulo_6.md [hunk 1]
```diff
@@ -136,46 +138,97 @@ Para esto vamos a agregar unha restricción de xeito que ese servicio só se exe
 
 _ \*Docker se caracteriza por traer baterías incluidas, pero intercambiables. Esto quere decir que hai partes do Docker Engine que, aínda que veñen cunha funcionalidade xa definida, se permiten intercambiar por outras de outro proveedor para mellorala. Principalmente hai 2 tipos de proveedores de plugins, os que proveen **plugins de rede**, e os que proveen **plugins para a xestión de volumenes**. Precisamente estos últimos están moi enfocados en tratar de atopar solucións para poder abstraer o volume de datos, do anfitrión, de xeito que se poida mover os contedores dos servicios entre nodos do cluster e continuen a ter os seus datos dispoñibles. Podedes ver algúns deles [aquí](https://store.docker.com/search?type=plugin). _
```
Comment (frmadem) [inline cursos/docker/00_actividades/06_modulo_6.md:139]: Faltarían comiñas para "baterías incluídas"

File: cursos/docker/00_actividades/06_modulo_6.md [hunk 2]
```diff
@@ -136,46 +138,97 @@ Para esto vamos a agregar unha restricción de xeito que ese servicio só se exe
...
+	3. Votacions
 
 Para o servicio de votacións, como é unha aplicación que non garda estado en si mesma, e está tendo moita demanda ;) vamos a escalalo a 2 replicas, de xeito que o cluster se encargue de balancear as peticións entre cada unha delas.
 
 Ademáis cando faga falta actualizar o servicio, queremos que se execute a actualización das 2 replicas á vez, e por suposto, se se cae que automáticamente se volva a arrincar soa.
 
-	1. Worker
+	4. Worker
 
 Para o nodo worker vamos a agregar 1 única réplica xa que a aplicación que nel corre non soporta máis, e admeáis agregarmoes unha política de restart on-failure.
```
Comment (frmadem) [inline cursos/docker/00_actividades/06_modulo_6.md:149]: typo "admeáis"

File: cursos/docker/00_actividades/06_modulo_6.md [hunk 3]
```diff
@@ -136,46 +138,97 @@ Para esto vamos a agregar unha restricción de xeito que ese servicio só se exe
...
 
-	1. Resultados:
+	5. Resultados:
 
 Para o nodo de resultados agregemos outras 2 réplicas, de maneira similar ao de votacions.
```
Comment (frmadem) [inline cursos/docker/00_actividades/06_modulo_6.md:153]: typo "agregemos"

**Review by frmadem — APPROVED** [review]

**Comment by alambike** [comment]
@yvidalv I think you missed linking the pull request with the issue so that it closes when you merge the PR.

You can do that with keywords like "Closes #xx": https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests

Can you add the link to the PR in the issue and close it manually?

**Pull request merged**